### PR TITLE
#163628455 Create database table and relation

### PR DESCRIPTION
**Pull request by EfeAgare**
#### What does this PR do?
Help creates the database table and relations for the application
#### How should this be manually tested?
After cloning the repo, cd into it, and RUN `npm install`
- setup your environment variable in .env file to connect to database
- run `npm run migrate`
#### What are the relevant pivotal tracker stories?
- [#163628455](https://www.pivotaltracker.com/story/show/163628455)

### CODE REVIEW

**Review by ozone4real — COMMENTED** [review]
Pull request title should be "Create database....." not "Creates"

**Comment by EfeAgare** [comment]
@ozone4real  Title has been fixed

**Review by ozone4real — DISMISSED** [review]

**Review by divinediscipline — COMMENTED** [review]
Is there a reason why the isVerified field is set to allow null value? The user should be verified before his details enter the database.

**Comment by EfeAgare** [comment]
@divinediscipline  isVerified field is set to allow null value? because we are the one to verify the user detail entrying the database, then setting isVerified to true

**Comment by omenkish** [comment]
> @divinediscipline isVerified field is set to allow null value? because we are the one to verify the user detail entrying the database, then setting isVerified to true

It should not allow null. It should be a boolean, default value false. Once a user is verified, the value becomes true

**Review by divinediscipline — DISMISSED** [review]
Looks good to me

**Review by bobbylemm — DISMISSED** [review]
looks good to me

**Review by biolabalo — DISMISSED** [review]
Looks Good To Me

File: server/db/migrations/20190131212909-create-follows.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    authorId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      references: {
+        model: 'Users',
+        key: 'id',
+        as: 'userId'
+      }
+    },
+    followId: {
```
Comment (omenkish) [inline server/db/migrations/20190131212909-create-follows.js]: @EfeAgare I think 'followerId' describes this attribute better than 'followId'.

**Review by primuse — DISMISSED** [review]
Looks good to me

File: server/db/models/users.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+      imageUrl: {
+        allowNull: true,
+        type: DataTypes.STRING
+      },
+      isVerified: {
+        defaultValue: false,
+        type: DataTypes.BOOLEAN
+      },
+      subscribed: {
+        allowNull: true,
```
Comment (omenkish) [inline server/db/models/users.js]: subscribed attribute should not allow null. It should have a default value of either true or false

**Review by primuse — COMMENTED** [review]
username should not be null. this is entered by the user while signing up.

**Review by p-funky — DISMISSED** [review]

**Review by ozone4real — DISMISSED** [review]
Looks good to me

**Review by omenkish — DISMISSED** [review]
Nice work @EfeAgare

**Review by primuse — CHANGES_REQUESTED** [review]
Username- allowNull should be false.

**Review by bobbylemm — APPROVED** [review]
this looks good to me

**Review by bobbylemm — CHANGES_REQUESTED** [review]
I believe article slug should be a string

**Review by omenkish — DISMISSED** [review]

**Review by biolabalo — DISMISSED** [review]

**Review by divinediscipline — DISMISSED** [review]
Looks good to me

**Review by primuse — DISMISSED** [review]

**Review by bobbylemm — DISMISSED** [review]

**Review by ozone4real — DISMISSED** [review]
LGTM

**Review by omenkish — APPROVED** [review]

**Review by biolabalo — APPROVED** [review]

**Review by ozone4real — APPROVED** [review]

**Review by bobbylemm — APPROVED** [review]
it looks good to me

**Review by divinediscipline — APPROVED** [review]
Looks good to me

**Review by primuse — APPROVED** [review]

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,2 +1,3 @@
 {
```
Comment (p-funky) [inline .vscode/settings.json:1]: Please add this folder directory `.vscode` to git ignore.

**Pull request merged**